stdlib/REPL/src/latex_symbols.jl

```diff
@@ -119,6 +119,44 @@ const latex_symbols = Dict(
     "\\euler" => "ℯ",
     "\\ohm" => "Ω",
 
+    # Music Symbols
```

Comment:
I'll just note that the norm here seems to be to have spaces around `=>`, it would be nice if this new code was consistent with that.

Also in the name of consistency, I think it would be good to keep the indentation of the comment char `#` to four spaces. To give the indication of nesting you want, perhaps try `# Music Symbols - Accidentals` etc.?